travis/install_grpc.sh

```diff
@@ -12,19 +12,19 @@ if [ "$grpc_dist" != "" ]; then
 fi
 
 # for python, we'll need the latest virtualenv and tox.
-# running gRPC requires the six package, any version will do.
+# running gRPC requires the six package, version >=1.10.
 if [ "$grpc_dist" != "" ]; then
-  pip install --upgrade --root $grpc_dist --ignore-installed virtualenv tox
-  pip install --root $grpc_dist --ignore-installed six
+  # Create a virtualenv, which also creates a virualenv-boxed pip.
+  virtualenv $grpc_dist/usr/local
+  $grpc_dist/usr/local/bin/pip install --upgrade --ignore-installed virtualenv tox six
```

Comment:
@michael-berlin found https://github.com/grpc/grpc/issues/5111#issuecomment-183489965 which recommends side-loading pip to avoid the apt dependency on the old version of python-six. That worked for our jessie-based docker image.